x-pack/plugins/actions/server/builtin_action_types/case/types.ts

```diff
@@ -122,7 +123,12 @@ export interface ExternalServiceApi {
 
 export interface CreateExternalServiceBasicArgs {
   api: ExternalServiceApi;
-  createExternalService: (credentials: ExternalServiceCredentials) => ExternalService;
+  createExternalService: (
+    credentials: ExternalServiceCredentials,
+    logger: Logger,
+    proxySettings?: any
```

Comment:
I've a got in this case `node plugins/security_solution/scripts/check_circular_deps` failing. I made it 'any', because when the Case team change their Jira, IBM resilient according to the ServiceNow example this code will be removed.